doc/GettingStarted.tex

```diff
@@ -319,21 +292,17 @@ \subsection{Run Buttons}
 \subsection{Zoom buttons}
 \label{ZoomButtons}
 
-\begin{center}
 \htmladdimg{NewItem17.png}
-\end{center}
 
 The Zoom buttons zoom in and out on the wiring canvas. The same functionality
 is embedded in the mouse scroll wheel. The reset zoom button
-\smhtmladdimg{zoomOrig.png} resets the zoom level to 1, and also
+\htmladdimg{zoomOrig.png} resets the zoom level to 1, and also
```

Comment:
Not sure about these changes. smhtmladdimg (small htmladdimg) and fwhtmladdimg (full width htmaddimg) dramatically improve the typset quality of the printed version.